drivers/SmartThings/matter-switch/src/aqara-cube/init.lua

```diff
@@ -7,15 +7,20 @@ local cubeFace = capabilities["stse.cubeFace"]
 
 local COMPONENT_TO_ENDPOINT_MAP_BUTTON = "__component_to_endpoint_map_button"
 local DEFERRED_CONFIGURE = "__DEFERRED_CONFIGURE"
-local DOING_CONFIGURE = "__DOING_CONFIGURE"
+
+-- used in unit testing, since device.profile.id and args.old_st_store.profile.id are always the same
+-- and this is to avoid the crash of the test case that occurs when try_update_metadata is performed in the device_init stage.
+local TEST_CONFIGURE = "__test_configure"
 local INITIAL_PRESS_ONLY = "__initial_press_only" -- for devices that support MS (MomentarySwitch), but not MSR (MomentarySwitchRelease)
 
 -- after 3 seconds of cubeAction, to automatically change the action status of Plugin UI or Device Card to noAction
 local CUBEACTION_TIMER = "__cubeAction_timer"
 local CUBEACTION_TIME = 3
 
 local function is_aqara_cube(opts, driver, device)
-  local name = string.format("%s", device.label)
+  -- device.label is a value that can be changed by the user,
+  -- so it is changed to read only value(device.manufacturer_info.product_name).
```

Comment:
nit: remove comment. Useful for us in PR, but not in the prod version

Reply:
I applied it.